### YamlMime:ManagedReference
items:
- uid: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException
  id: ForeignKeyReferenceAlreadyHasValueException
  children:
  - System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.#ctor
  - System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.#ctor(System.String)
  - System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: ForeignKeyReferenceAlreadyHasValueException
  nameWithType: ForeignKeyReferenceAlreadyHasValueException
  fullName: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException
  type: Class
  summary: "Represents errors that occur when an attempt is made to change a foreign key when the entity is already loaded."
  syntax:
    content: 'public class ForeignKeyReferenceAlreadyHasValueException : InvalidOperationException'
  inheritance:
  - System.InvalidOperationException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.#ctor
  id: '#ctor'
  parent: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException
  langs:
  - csharp
  name: ForeignKeyReferenceAlreadyHasValueException()
  nameWithType: ForeignKeyReferenceAlreadyHasValueException.ForeignKeyReferenceAlreadyHasValueException()
  fullName: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.ForeignKeyReferenceAlreadyHasValueException()
  type: Constructor
  assemblies:
  - System.Data.Linq
  namespace: System.Data.Linq
  summary: "Initializes a new instance of the <xref href=&quot;System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException&quot;></xref> class with a system-supplied message that describes the error."
  remarks: "This constructor initializes the <xref:System.Exception.Message%2A> property of the new instance to a system-supplied message that describes the error, such as &quot;DefaultMessageDisplayedByParameterlessCtorWriterMustSupply&quot;. This message takes into account the current system culture.       The following table shows the initial property values for an instance of <xref:System.InsufficientMemoryException>.      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|`null`.|   |<xref:System.Exception.Message%2A>|The localized error message string.|"
  syntax:
    content: public ForeignKeyReferenceAlreadyHasValueException ();
    parameters: []
  overload: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException
  langs:
  - csharp
  name: ForeignKeyReferenceAlreadyHasValueException(String)
  nameWithType: ForeignKeyReferenceAlreadyHasValueException.ForeignKeyReferenceAlreadyHasValueException(String)
  fullName: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.ForeignKeyReferenceAlreadyHasValueException(String)
  type: Constructor
  assemblies:
  - System.Data.Linq
  namespace: System.Data.Linq
  summary: "Initializes a new instance of the <xref href=&quot;System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException&quot;></xref> class with a specified message that describes the error."
  remarks: "The following table shows the initial property values for an instance of <xref:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException>.      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|`null`.|   |<xref:System.Exception.Message%2A>|The error message string specified in `message`.|"
  syntax:
    content: public ForeignKeyReferenceAlreadyHasValueException (string message);
    parameters:
    - id: message
      type: System.String
      description: "The message that describes the exception. The caller of this constructor is required to ensure that this string has been localized for the current system culture."
  overload: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException
  langs:
  - csharp
  name: ForeignKeyReferenceAlreadyHasValueException(String,Exception)
  nameWithType: ForeignKeyReferenceAlreadyHasValueException.ForeignKeyReferenceAlreadyHasValueException(String,Exception)
  fullName: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.ForeignKeyReferenceAlreadyHasValueException(String,Exception)
  type: Constructor
  assemblies:
  - System.Data.Linq
  namespace: System.Data.Linq
  summary: "Initializes a new instance of the <xref href=&quot;System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException&quot;></xref> class with a specified error message and a reference to the inner exception that is the cause of this exception."
  remarks: "An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property. The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or `null` if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.       The following table shows the initial property values for an instance of <xref:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException>.      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|`null`.|   |<xref:System.Exception.Message%2A>|The error message string specified in `message`.|"
  syntax:
    content: public ForeignKeyReferenceAlreadyHasValueException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "The message that describes the exception. The caller of this constructor is required to ensure that this string has been localized for the current system culture."
    - id: innerException
      type: System.Exception
      description: "The exception that is the cause of the current exception. If the `innerException` parameter is not <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;></xref>, the current exception is raised in a <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;></xref> block that handles the inner exception."
  overload: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.InvalidOperationException
  isExternal: false
  name: System.InvalidOperationException
- uid: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.#ctor
  parent: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException
  isExternal: false
  name: ForeignKeyReferenceAlreadyHasValueException()
  nameWithType: ForeignKeyReferenceAlreadyHasValueException.ForeignKeyReferenceAlreadyHasValueException()
  fullName: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.ForeignKeyReferenceAlreadyHasValueException()
- uid: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.#ctor(System.String)
  parent: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException
  isExternal: false
  name: ForeignKeyReferenceAlreadyHasValueException(String)
  nameWithType: ForeignKeyReferenceAlreadyHasValueException.ForeignKeyReferenceAlreadyHasValueException(String)
  fullName: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.ForeignKeyReferenceAlreadyHasValueException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.#ctor(System.String,System.Exception)
  parent: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException
  isExternal: false
  name: ForeignKeyReferenceAlreadyHasValueException(String,Exception)
  nameWithType: ForeignKeyReferenceAlreadyHasValueException.ForeignKeyReferenceAlreadyHasValueException(String,Exception)
  fullName: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.ForeignKeyReferenceAlreadyHasValueException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.#ctor*
  parent: System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException
  isExternal: false
  name: ForeignKeyReferenceAlreadyHasValueException
  nameWithType: ForeignKeyReferenceAlreadyHasValueException.ForeignKeyReferenceAlreadyHasValueException
